src/routes/api/tasks/index.ts

```diff
@@ -334,16 +334,20 @@ const plugin: FastifyPluginAsyncTypebox = async (fastify) => {
           filename
         )
 
-        await fs.promises.unlink(filePath)
+        await fs.promises.unlink(filePath).catch((err) => {
+          // Just warn if file not found as file deletion 
+          // should be treated as successful if the database operation succeeds,
```

Comment:
```suggestion
          // A file could have been deleted by an external actor, e.g. system administrator.
          // We log the error to keep a record of the failure, but consider that the operation was successful.
```

Use directly a try/catch instead of `.catch`, I was wrong to offer to handle this way.